tools/macos-installer/productbuild/Resources/en.lproj/welcome.html.tmpl

```diff
@@ -0,0 +1,15 @@
+<html>
+ <head>
+    <style>
+      body {
+        font-family: Helvetica;
+        font-size: 1.1em;
+      }
+    </style>
+ </head>
+ <body>
+  <p>
+    This package will install Node.js {nodeversion} and npm {npmversion} into <code>/usr/local/</code>.
```

Comment:
It would be nice if this message is formatted similar to how the conclusion message is formatted, maybe like this:

```html
<div>
  <p>This package will install</p>
  <ul>
    <li>Node.js {nodeversion} to <code>/usr/local/bin/node</code></li>
    <li>npm {npmversion} to <code>/usr/local/bin/npm</code></li>
  </ul>
</div>
```

Reply:
How about this? <img width="732" alt="screen shot 2017-09-07 at 08 29 17" src="https://user-images.githubusercontent.com/1814479/30149338-8ceeed5e-93a7-11e7-9f10-a3500f8a44df.png">

Reply:
Looking good!